checker.go

```diff
@@ -364,6 +364,50 @@ func (c *hasLenChecker) Check(got interface{}, args []interface{}, note func(key
 	return nil
 }
 
+// Implements checks that the provided value implements the given interface. A
+// pointer to the interface must be provided.
+//
+// For instance:
+//
+//     c.Assert(myReader, qt.Implements, io.ReadCloser)
+//
+var Implements Checker = &implementsChecker{
+	argNames: []string{"got", "want"},
```

Comment:
"want interface"? I know it's not used by this checker, but could be good for documentation purposes.

Reply:
I think `want interface pointer` reads nicely, good point.

Reply:
Updated to `want interface pointer`. Thanks!